src/realtime-callbacks.js

```diff
@@ -56,6 +56,91 @@ module.exports.setNow = function(f) {
 };
 let _now = Date.now;
 
+// Internal hash of active debouncers
+const _debouncers = {};
+
+class Debouncer {
+
+    constructor(jobName) {
+        this.jobName = jobName;
+    }
+
+    complete() {
+        delete _debouncers[this.jobName];
+        this.func();
+    }
+
+    start(func, waitMs) {
+        if (func) this.func = func;
+        if (!isNaN(waitMs)) this.waitMs = waitMs;
```

Comment:
This won't do what you want it to do.
```js
> isNaN(null)
false
```

Meaning `waitMs` could be set to `null`. You really only want positive `waitMs` values, so just do:

```js
if (waitMs > 0) {
    this.waitMs = waitMs;
}
```